aspnetcore/mvc/views/tag-helpers/built-in/samples/TagHelpersBuiltIn/Pages/Products.cshtml

```diff
@@ -0,0 +1,21 @@
+﻿@page
+@model TagHelpersBuiltIn.Pages.ProductsModel
+@{
+    ViewData["Title"] = "Products";
+}
+
+<h2>Products</h2>
+
+<!-- <snippet_HtmlHelper> -->
+@foreach (var product in Model.Products)
+{
+    @await Html.PartialAsync("_ProductPartial", product)
+}
+<!-- </snippet_HtmlHelper> -->
+
+<!-- <snippet_TagHelper> -->
+@foreach (var product in Model.Products)
+{
+    <partial name="_ProductPartial" model="product" />
+}
+<!-- </snippet_TagHelper> -->
```

Comment:
A return here to make GH happy. :smiley:

Reply:
We should discuss using *.editorconfig* to enforce conventions like this repo-wide.

Reply:
We've done a good job AFAICT. The only sore spots are the *wwwroot* assets provided from external sources, the *lib* folder. They're **not good** on this point. However ... however ... I can *script kitty* us to consistency if you like. Let me know. The only concern there is new *lib* folders coming in ... we'd each need to make sure our "stock" *lib* folders for topics all have the same treatment.

Reply:
https://github.com/aspnet/Docs/issues/7489